packages/support/resources/views/components/button/index.blade.php

```diff
@@ -62,9 +64,9 @@
             // @deprecated `fi-btn-color-*` has been replaced by `fi-color-*` and `fi-color-custom`.
             is_string($color) ? "fi-btn-color-{$color}" : null,
             is_string($color) ? "fi-color-{$color}" : null,
-            "fi-size-{$size->value}" => $size instanceof ActionSize,
+            "fi-size-{$sizeClassSuffix}" => $size instanceof ActionSize,
```

Comment:
Could you update the PR to just use the value inline? No need to have the array value as condition if we already conditionally return the string or null.